netlify.toml

```diff
@@ -1,2 +1,5 @@
 [build]
   Functions = "./functions"
+
+[context.deploy-preview.environment]
+  GATSBY_ENVIRONMENT = "preview"
```

Comment:
I'm using this `GATSBY_ENVIRONMENT` env var to check wether I should include some `dev` code, such as the GridDebugger. We may use this for other stuff in the future, such as enabling/disabling sourcemaps, etc

Reply:
🎉

Reply:
💃